### ISSUE TITLE
temporarily uploaded images need to be deleted

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/kiosk#76]
I forgot to delete the images that have been uploaded for import after they have been processed. Important! They will accumulate in the temp directory.

**Comment by urapadmin** [comment]
also files in the normal temp directory need to be cleaned up (see #141). MCP should do that job.

**Comment by urapadmin** [comment]
it is not that simple. MCP is project-independent and servers several projects at the same time. Currently I think it would need something like a regularly running garbage-collector job that is started by each project - but only once! 
That would need a couple of new features in MCP. But it is the only really safe way to do it. 
The problem is not imminent, so I will postpone this part. 

What I will do is create the mechanism that removes files from a directory if they are above a certain age and call it after single file import.

**Comment by urapadmin** [comment]
### kiosk 0.8.27
after uploading an image, no matter if it is successful or not, both the file_repository\temp and kiosk\temp directories are cleaned up of all files older than 30 days.

**Comment by urapadmin** [comment]
this seems to be working by and large. At least I cannot see a file from 2021 anymore and I can check again on the 21.II.2022 to see if the 1GB.bin file is gone from the file_repository under test.

**Comment by urapadmin** [comment]
I tested it on urap: works.
